### DISCUSSION TITLE
File transfer using Cyberduck

### DISCUSSION THREAD

**Discussion by beliven-fabrizio-gortani** [gravitational/teleport#29818] [Q&A] [answered]
Hi everyone.
There is a guide to setup [Cyberduck](https://cyberduck.io/) (file transfer client) for MacOS for use with Teleport? I've try different configurations to configure smtp without success. 
Do you have any idea?

**Comment by webvictim** [comment, marked as answer]
I don't believe this will ultimately be possible as it doesn't appear that Cyberduck supports SSH host certificates.

Their docs (https://docs.cyberduck.io/protocols/sftp/) do indicate that SSH user certificates signed by a CA will work and that they support the use of an SSH `ProxyCommand`, but I haven't been able to get this to work properly with Teleport in any of the testing I've done. They seem to skip running the `ProxyCommand` and just try to connect directly to the server (which won't work)

**Comment by beliven-fabrizio-gortani** [comment]
Thanks for the answer @webvictim 

I'm try to explain my problem.

In the company I work for, we use Teleport to connect to our machines. For the internal team's workflow, there are no issues, as every project is released through a Git pipeline. However, some projects, especially Wordpress sites, involve external individuals who typically use Windows or Mac for edit some files directly. We were looking for an SFTP client that works with Teleport for these platforms.

I've read also this [discussion](https://github.com/gravitational/teleport/discussions/8823). 
I have seen that VSCode could be a good option, but I wanted to understand if there are any pure SFTP clients that support Teleport

**Reply by webvictim** [reply]
There aren't any that I know of yet. However, I believe WinSCP is adding support for host certificates (https://winscp.net/tracker/2145) and will likely be the first client to support connecting to Teleport hosts when that support is added. I don't have any timeline for that at the moment, however.
